src/app/pages/event/create.vue

```diff
@@ -11,23 +132,290 @@
 </template>
 
 <script setup lang="ts">
+import { ref, computed } from 'vue'
+import { storeToRefs } from 'pinia'
+import { useI18n } from 'vue-i18n'
+import { useEventForm } from '~/composables/useEventForm'
+import { useCreateEventMutation } from '~~/gql/documents/mutations/event/eventCreate'
+import { useUploadCreateMutation } from '~~/gql/documents/mutations/upload/uploadCreate'
+import { Uppy } from '@uppy/core'
+
+import { EventVisibility } from '~~/gql/generated/graphql'
+import Tus from '@uppy/tus'
+import type { EventStorageStrategy } from '~/composables/storage/EventStorageStrategy'
+import { LocalStorageStrategy } from '~/composables/storage/LocalStorageStrategy'
+
```

Comment:
Dependencies first, own content second

```suggestion
import { Uppy } from '@uppy/core'
import Tus from '@uppy/tus'
import { storeToRefs } from 'pinia'

import { EventVisibility } from '~~/gql/generated/graphql'
import { useEventForm } from '~/composables/useEventForm'
import { useCreateEventMutation } from '~~/gql/documents/mutations/event/eventCreate'
import { useUploadCreateMutation } from '~~/gql/documents/mutations/upload/uploadCreate'
import type { EventStorageStrategy } from '~/composables/storage/EventStorageStrategy'
import { LocalStorageStrategy } from '~/composables/storage/LocalStorageStrategy'

```

Not sure if `storeToRefs` is needed